packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/FileSystemGraphObjectStore.ts

```diff
@@ -18,37 +18,31 @@ import {
   iterateRelationshipTypeIndex,
   readGraphObjectFile,
 } from './indices';
+import { InMemoryGraphObjectStore } from '../memory';
 import { FlushedEntityData } from '../types';
 import { getRootStorageAbsolutePath } from '../../fileSystem';
 import { BigMap } from '../../execution/utils/bigMap';
-import { chunk, min } from 'lodash';
-import { InMemoryGraphObjectStore } from '../memory';
+import { chunk } from 'lodash';
 
+export const DEFAULT_GRAPH_OBJECT_BUFFER_THRESHOLD = 500;
 export const DEFAULT_GRAPH_OBJECT_FILE_SIZE = 500;
 
-export const DEFAULT_GRAPH_OBJECT_BUFFER_THRESHOLD_IN_BYTES = 5_000_000;
-// no more than 10^9 bytes
-export const MAX_GRAPH_OBJECT_BUFFER_THRESHOLD_IN_BYTES = 1_000_000_000;
 // it is important that this value is set to 1
 // to ensure that only one operation can be performed at a time.
 const BINARY_SEMAPHORE_CONCURRENCY = 1;
 
 export interface FileSystemGraphObjectStoreParams {
   integrationSteps?: IntegrationStep[];
-  /**
-   * The maximum size in bytes of entities/relationships stored in memory at one time.
-   * default: 25_000_000
-   */
-  graphObjectBufferThresholdInBytes?: number;
+
```

Comment:
This is a public interface, so we should not remove any arguments. We'll just leave this property unused for now.

```suggestion
  /**
   * The maximum size in bytes of entities/relationships stored in memory at one time.
   * default: 25_000_000
   */
  graphObjectBufferThresholdInBytes?: number;
```